bundles/org.openhab.core.addon.marketplace/src/main/java/org/openhab/core/addon/marketplace/internal/community/CommunityMarketplaceAddonService.java

```diff
@@ -86,17 +90,20 @@
     private static final String YAML_CODE_MARKUP_START = "<pre><code class=\"lang-yaml\">";
     private static final String CODE_MARKUP_END = "</code></pre>";
 
-    private HashMap<Integer, AddonType> types = new HashMap<Integer, AddonType>(3);
-    private static final Integer BINDINGS_CATEGORY = 73;
+    private static final Integer ADDONS_CATEGORY = 73;
```

Comment:
Rule templates and UI Widgets are also add-ons, these are bundle add-ons.

```suggestion
    private static final Integer BUNDLES_CATEGORY = 73;
```

Reply:
done